frontend/src/components/forms/CharacterForm.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+
+const CharacterForm = () => {
```

Comment:
The problem we have in this code is that we are generating a new form component every time the drop down menu changes which goes against the logic of using react in the first place.

What we can do is create a generic dropdown form that we can use repeatedly. We can effectively pass in the items that will appear in the drop down menu and what type of choice the user is making through props (for example: type of choice - character, available options: spiderman, rapunzel etc..)

Comments on this file is suggested this for this particular form, but I think this logic can be used to generate all the other forms (effectively making the others redundant)

`
const Form = ({dropdownItems, selectionField}) = () => {
const [dropdown, setDropdown] = useState(dropdownItems)
`